### ISSUE TITLE
KingStar, you are the author of the modification Restools-Inno Setup?(apologies for my English, translated by google translator.)

### ISSUE THREAD

**Issue by nia1nia1** [KngStr/Inno-All-in-One-Setup#7]

**Comment by KngStr** [comment]
I'm sorry I'm not.
The author is over here: http://restools.hanzify.org/

**Comment by nia1nia1** [comment]
Thanks for the answer. I'm looking for the author of this masterpiece, I want to ask him to share the source code. But as I understand the site does not work, there will be no updates and there is no connection with the author.I need to swap the pages in the installer so that first there is a page for selecting components, and then a page for selecting the installation folder. In the original Inno Setup 6.0.3, I did this without difficulty, swapping places in the code of Setup.e32. But my script was created in the modified version, the voice acting of the buttons on mouseover, RawDataResource, which is not available in the original version. So I was hoping that I would find the source code and modify it for myself.I found the site in the cache.  There are even working download links. But I can’t find any contact details there.http://web.archive.org/web/20150315014015/http://restools.hanzify.org/07.12.2019, 11:54, "KngStr" <notifications@github.com>:I'm sorry I'm not.
The author is over here: http://restools.hanzify.org/
